doc/integrator/querier.rst

```diff
@@ -25,16 +25,16 @@ The web service configuration is done in the ``vars_<project>.yaml`` file:
                         separator: ","
 
 The ``dbsession: "<session name>"`` in the ``enum`` is just a shortcut
-if almost of the session are similar in the layers. If the dbsession isn't
-defined we use the main DB session.
+if almost of the session are similar in the layers. If ``dbsession`` is not
+defined, the main DB session is used.
 
 The ``table: "<[schema.]table name>"`` in the layer is just a shortcut
-if almost of the attribute of the layer are in the same table.
-Each attributes requires a table.
+if almost of the attributes of the layer are in the same table.
```

Comment:
Maybe something like:

``table: "<[schema.]table name>"`` may be used at the layer level as a the default table where following attributes may be found. It can be overridden at the attribute level.